manifest.json

```diff
@@ -13,7 +13,16 @@
     "48": "icon.png",
     "64": "icon64.png"
   },
-
+  "experiment_apis": {
```

Comment:
What will we do if this new API isn't available, like if we try to install it in Firefox Release, for instance?